src/actor/Claimer.ts

```diff
@@ -40,6 +45,7 @@ function finalizeReqProps(props: string[]): string[] {
  * @param credentials The [[Credential]]s which should be verified.
  * @param attesterPubKeys The privacy enhanced public keys of all [[AttesterIdentity]]s which signed the [[Credential]]s.
  * @param requirePE A boolean to force privacy enhancement.
+ * @throws ERROR_PE_MISMATCH, ERROR_MESSAGE_TYPE, ERROR_PE_CREDENTIAL_MISSING.
```

Comment:
It would be better to put each of these into square brackets `[[...]]` so that they link to the error in the API documentation.
<img width="618" alt="Screenshot 2020-06-18 at 13 25 58" src="https://user-images.githubusercontent.com/26296386/85014818-42483e00-b167-11ea-8dc7-9bfd0dd55c8c.png">

Reply:
Yes, thanks for the heads up, I wasn't sure if formatting would work on these.

This is great!!!

Reply:
finding them all is quite the headache :sweat: